projects/Mallard/android/app/build.gradle

```diff
@@ -179,6 +179,8 @@ def appVersionName() {
     return majorVersion + "." + minorAppVersion() + "." + buildNumber()
 }
 
+println("::set-output name=VERSION_NAME::${appVersionName()}")
```

Comment:
Do we need this one? Or was this for our testing?

Reply:
This is a command to the github action environment to set a variable called `VERSION_NAME`. It is referenced in the tag command as `git tag v${{ steps.build_app.outputs.VERSION_NAME }}`. There is also an equivalent for `VERSION_CODE`